Source/Core/VideoCommon/FramebufferManager.cpp

```diff
@@ -874,6 +883,8 @@ void FramebufferManager::DestroyClearPipelines()
 
 void FramebufferManager::PokeEFBColor(u32 x, u32 y, u32 color)
 {
+  LWDProfiler::ScopedMarker marker("FBuffer/Poke EFB Color");
```

Comment:
Most this marker to FlushEFBPokes, which is the actual expensive operation (and combine with depth)